tools/cbmc/README.md

```diff
@@ -55,6 +55,11 @@ Installing CBMC
 Setting up the proofs
 ---------------------
 
+Change into the `patches` directory. Run
```

Comment:
@karkhaz

These lines are added to address your comment. As you say this is not sufficient, could you describe in more detail, what kind of user documentation enhancement you envision?

Reply:
Thanks, I completely missed this for some reason!